operator/cmd/run/run.go

```diff
@@ -229,7 +209,6 @@ func Run(
 	kube.ManagedFieldsManager = controllerName
 
 	mgrOptions := ctrl.Options{
-		Scheme:                  scheme,
```

Comment:
Should we also expose a unified scheme to make the setup here a bit more simple? Very puntable.